releng/devkit.py

```diff
@@ -528,17 +529,26 @@ def deduplicate(items):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 4:
-        print("Usage: {0} kit host outdir".format(sys.argv[0]), file=sys.stderr)
-        sys.exit(1)
-
-    kit = sys.argv[1]
-    host = sys.argv[2]
-    outdir = os.path.abspath(sys.argv[3])
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument("kit")
+    parser.add_argument("host")
+    parser.add_argument("outdir")
+    parser.add_argument("-t", "--thin", help='build without cross-arch support', action="store_true")
```

Comment:
```suggestion
    parser.add_argument("-t", "--thin", help="build without cross-arch support", action='store_true')
```

(Should use double quotes for regular strings, and single quotes for enum/symbol values. In hindsight this is a bit of a weird convention, but better stick to it for consistency and consider changing it globally later.)

Reply:
Oops, good catch!